lib/tasks/publish.rake

```diff
@@ -0,0 +1,13 @@
+# This tasks will publish documents that are in a draft publication state.
+
+require "services"
+
+namespace :publish do
+  desc "Publishes all documents in a draft publication state.\n" \
+    "Usage: `rake publish:all` or with types `rake publish:all protected_food_dring_name esi_fund`"
+  task :all, [:document_types] => :environment do |_, args|
+    types = args.document_types.presence&.split(" ")
```

Comment:
```suggestion
    types = args.extras
```

I'd suggest doing this then the invocation is `rake publish:all[protected_food_drink_name,esi_find]` which feels more like a traditional Rake task.